sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1425 @@
+        externalId: workflowName
+        type: String
+      - name: identityRequestId
+        externalId: identityRequestId
+        type: String
+      - name: workflowCaseId
+        externalId: workflowCaseId
+        type: String
+      - name: workflowProcessId
+        externalId: workflowProcessId
+        type: String
+      - name: retries
+        externalId: retries
+        type: Int64
+      - name: approvalSet
+        externalId: approvalSet
+        type: String
+      - name: workflowSummary
+        externalId: workflowSummary
+        type: String
+      - name: targetClass
+        externalId: targetClass
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: messages
+        externalId: $.messages[*].[*] # Flatten a [][]string to a []string
+        type: String
+        list: true
+      - name: completionStatus
+        externalId: completionStatus
+        type: String
+      - name: taskDefinition
+        externalId: taskDefinition
+        type: String
+      - name: terminated
+        externalId: terminated
+        type: Bool
+      - name: launcher
+        externalId: launcher
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  LaunchedWorkflowInput:
+    parent: LaunchedWorkflows
+    displayName: SailpointLaunchedWorkflowInput
+    externalId: input
+    description: Entity representing the inputs to a Sailpoint Identity IQ Launched Workflow.
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: key
+        externalId: key
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: value
+        externalId: value
+        type: String
+      - name: type
+        externalId: type
+        type: String
+
+  LaunchedWorkflowOutput:
+    parent: LaunchedWorkflows
+    displayName: SailpointLaunchedWorkflowOutput
+    externalId: output
+    description: Entity representing the outputs to a Sailpoint Identity IQ Launched Workflow.
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: key
+        externalId: key
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: value
+        externalId: value
+        type: String
+      - name: type
+        externalId: type
+        type: String
+
+  LaunchedWorkflowAttribute:
+    parent: LaunchedWorkflows
+    displayName: SailpointLaunchedWorkflowAttributes
+    externalId: attributes
+    description: Entity representing the Sailpoint alert attributes each user is associated with.
```

Comment:
Is this description correct? The API docs list `A list of attributes of the TaskResult. (From the TaskResult used to launch the Workflow).`

https://developer.sailpoint.com/iiq/api/launched-workflows-get